### ISSUE TITLE
Naive Program - devices get stuck

### ISSUE THREAD

**Issue by Philippe-Drolet** [coreylammie/MemTorch#57]
Hello, 

The updated naive program does not work for my network (a simple 1-layer (linear) MNIST NN)) both when using the default parameters for VTEAM and the parameters representing our team's devices (in Data_driven). No error is raised since the program reaches lines 124 and 125 of the Program.py file:
            print(crossbar.devices[point].g)
            exit(0)
Is it normal to be unable to program a crossbar to represent certain weight matrices? This is true with or without tiled crossbars. Thank you,

Philippe Drolet

**Comment by coreylammie** [comment]
Hi @Philippe-Drolet,

Thank you for pointing this out! 

I can confirm that these two lines were left-over from my original debugging efforts, and should be removed. Consequently, I have removed them in [naive_program](https://github.com/coreylammie/MemTorch/tree/naive_program), which will be merged into `master` later today. I'll add additional unit test(s) for stuck devices during programming to catch any future semantic errors- it appears that no devices encountered the memristor boundary lock problem when unit tests were originally run! I'll link the corresponding pull request to this issue so you will be notified once it is merged.

Given an arbitrary weight matrix of the correct shape, `memtorch.bh.Crossbar.write_conductance_matrix` can be used to program a crossbar (or set of tiles)- the behaviour you are experiencing is simply caused by the erroneous `exit(0)` statement.

That aside, I intend on adding additional input arguments to `naive_program` specifying (1) the relative tolerance threshold between high and low conductance states to the stuck conductance state, and (2) optional voltage thresholds; both to force shift a device's conductance when the memristor boundary lock problem is encountered. 

Kind Regards,

Corey.
